packages/devtools/lib/src/vm_service_wrapper.dart

```diff
@@ -252,6 +253,10 @@ class VmServiceWrapper implements VmService {
   Future<Version> getVersion() =>
       _trackFuture('getVersion', _vmService.getVersion());
 
+  @override
+  Future<dynamic> getMemoryUsage(String isolateId) =>
```

Comment:
@terrylucas, fyi for the new memory usage API (this returns a `MemoryUsage` class).